packages/debugger/lib/debugger.js

```diff
@@ -28,15 +29,22 @@ const Debugger = {
    * @return {Debugger} instance
    */
   forTx: async function (txHash, options = {}) {
-    let { contracts, files, provider, compilations, lightMode, storageLookup } =
-      options;
+    const {
+      contracts,
+      files,
+      provider,
+      compilations,
+      lightMode,
+      storageLookup,
+      ensRegistryAddress
```

Comment:
Can we make an `ens` namespace here and make `registryAddress` a sub-field of that? I have a strong feeling we'll want to add things here later.

Reply:
Sure, that's entirely doable.